packages/wonder-blocks-dropdown/src/components/option-item.js

```diff
@@ -190,7 +190,7 @@ const {blue, white, offBlack, offBlack32} = Color;
 const styles = StyleSheet.create({
     itemContainer: {
         flexDirection: "row",
-        backgroundColor: white,
+        background: white,
```

Comment:
This is unrelated to these changes, but I changed this so Jest stops complaining about this:

```
 console.error
    Warning: Removing a style property during rerender (background) when a conflicting property is set (backgroundColor) can lead to styling bugs. To avoid this, don't mix shorthand and non-shorthand properties for the same value; instead, replace the shorthand with separate values.
        in button (created by StyleComponent)
        in StyleComponent (created by ClickableBehavior)
        in ClickableBehavior (created by Context.Consumer)
        in SelectOpener (created by SingleSelect)
        in div (created by StyleComponent)
        in StyleComponent (created by View)
        in View (created by DropdownCore)
        in DropdownCore (created by ActionSchedulerProvider)
        in ActionSchedulerProvider (created by ForwardRef)
        in ForwardRef (created by SingleSelect)
        in SingleSelect (created by BirthdayPicker)
        in div (created by StyleComponent)
        in StyleComponent (created by View)
        in View (created by BirthdayPicker)
        in BirthdayPicker
```